### PR TITLE
Migration 2to3

### PR DESCRIPTION
**Pull request by sklencar**

### CODE REVIEW

File: debuggerwidget.py [hunk 1]
```diff
@@ -383,7 +393,7 @@ def on_load(self):
         settings = QSettings()
         folder = settings.value("firstaid/lastFolder", '')
 
-        filename = QFileDialog.getOpenFileName(self, "Load", folder, "Python files (*.py)")
+        filename, __ = QFileDialog.getOpenFileName(self, "Load", folder, "Python files (*.py)")
```
Comment (wonder-sk) [inline debuggerwidget.py]: At least in QGIS 2.x this results in a Python error: ValueError: too many values to unpack

File: sourceview.py [hunk 2]
```diff
@@ -52,7 +56,7 @@ def __init__(self, parent=None):
         # override style settings to the same font and size
         # (python lexer has styles 0 ... 15)
         for i in range(16):
-          self.SendScintilla(QsciScintilla.SCI_STYLESETFONT, i, fontName)
+          #self.SendScintilla(QsciScintilla.SCI_STYLESETFONT, i, fontName)
```
Comment (wonder-sk) [inline sourceview.py]: why disable this?

Comment (sklencar) [reply sourceview.py]: Some problem with Scintillia, not anymore -> enabled again.

File: debuggerwidget.py (hunk 1 above)
Comment (sklencar) [reply debuggerwidget.py]: Replicated when no file has been chosen. Fixed, probably some wrong fix of https://github.com/qgis/QGIS/blob/master/scripts/2to3

**Comment by sklencar** [comment]
Cannot replicate a problem with console when an error occurs.

**Pull request merged**